spectral_cube/spectral_cube.py

```diff
@@ -2519,8 +2519,10 @@ def spectral_interpolate(self, spectral_grid,
             cubedata = self.filled_data[::-1]
```

Comment:
@keflavich - Does the whole cube need to be loaded in like this? Is there an advantage to doing so?